config/sync/core.entity_view_display.paragraph.stanford_lists.default.yml

```diff
@@ -31,7 +31,7 @@ content:
       target: '0'
     third_party_settings:
       field_formatter_class:
-        class: su-margin-top-5
+        class: ''
```

Comment:
What happens if the list is empty? does the buttom push itself to close to the description/heading?

Reply:
Oh good point. I did test with the Heading + Description + Button and Description + Button. But not Heading + Button. I made a page of examples to show Design and I'll tag you too.

Reply:
@pookmish and @imonroe I spoke with Ishi and she was good with the spacing. I put screenshots of the display in the [ticket](https://stanfordits.atlassian.net/browse/D8CORE-4093) with the different scenarios. I am going to make another ticket for the other spacing issue she mentioned.